code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -480,18 +480,27 @@
 	process_flags = ORGANIC | SYNTHETIC
 	
 /datum/reagent/mutationtoxin/lizard/on_mob_life(mob/living/carbon/human/H)
```

Comment:
nie podoba mi się że ta toksyna która niby ma zamieniać w lizarda nagle może zamieniać we wszystkie rasy, daj to do jakiejś bardziej ogólnej toksyny, albo stwórz nową, ta jest opisana jako toksyna zmieniająca w lizarda

Reply:
a tak było wcześniej. że dajesz radium i masz nową toskynę

Reply:
? mi chodzi o to że jest wiele mutationtoxin, a on dał tych wiele ras akurat do lizardowej

Reply:
Tak było robione na TG, Joker chciał by zamiast Lizarda po dodaniu radium miało zmienić Cię w jednego z wymienionych: mush, snail, vampire, zombie i skeleton

Reply:
Ja miałem tylko robić port wcześniejszych rzeczy, stworzenie nowej toksyny może nie być takie banalne jak zmienienie jednej z nich.

Reply:
Ja nie widzę problemu, bo nigdy w życiu nie widziałem, żeby ktoś się zmieniał w lizarda

Reply:
https://github.com/aq33/tgstation/blob/07dfef9dae8ce677238424b650617b280ce6e293/code/modules/reagents/chemistry/reagents/other_reagents.dm#L449
a to? czasem to nie robi tego co ty chcesz dodać? skąd ten toxin jest? jakoś ze slimeami związane? bo jeżeli tak to wystarczy tam podmienić z tego lizarda na unstable

Reply:
z tych slimeów otrzymuje się unstable toxin który robi to czego chcecie https://github.com/aq33/tgstation/blob/221d9afa67b8748d036d8a29117654cdefc6df6f/code/modules/reagents/chemistry/recipes/slime_extracts.dm#L60
unstable toxin może właśnie w jedną z wielu ras zmienić tam losowo
inne toxins są robione z unstable toxin i pozwalają na lepszą kontrolę nad tym w co się zmienisz

Reply:
Wychodzi na to, że dostaje się go zamiast radium, najwyraźniej na bee nie da się w lizarda zmienić.

Reply:
da się, tylko trzeba ten lizard toxin z tego unstable sobie zrobić czymś tam, a to co chcesz dodać już najwidoczniej jest w bee, więc wywal tą twoją część z lizard toxinem

Reply:
wywalone